src/types/types.ts

```diff
@@ -312,7 +322,7 @@ export interface WalletListItem {
 
   // These will be set for token rows:
   token?: EdgeToken
-  tokenId?: string
+  tokenId: EdgeTokenId
```

Comment:
Not sure I understand. The comment preceeds the `token?: EdgeToken` line which is still true in that the `edgeToken` object will only be set for token rows. Maybe change comment to `// This will be set for token rows`